redi-connect-front/src/components/organisms/Navbar.tsx

```diff
@@ -2,16 +2,44 @@ import React, { useState } from 'react'
 import Button from '../atoms/Button'
 import { isLoggedIn } from '../../services/auth/auth'
 import { logout } from '../../services/api/api'
-import { Section, Container, Element } from 'react-bulma-components'
+import { Section, Container, Element, Content } from 'react-bulma-components'
 import Logo from '../atoms/Logo'
 import classnames from 'classnames'
-import { useHistory } from 'react-router-dom'
+import { useHistory, NavLink } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
 import './Navbar.scss'
 
+interface NavItemProps {
+  url: string
+  children: any
+}
+
+const LoggedOutNavItems = () => {
+  const { t } = useTranslation()
+
+  const NavItem = ({ url, children }: NavItemProps) => (
```

Comment:
Check if we could tune the button to be rendered as Navlink. Would be a nicer solution .